source/tutorials.rst

```diff
@@ -264,3 +372,5 @@ References
 .. bibliography:: WEC-Sim_Tutorials.bib
    :style: unsrt   
 
+.. |tutorialspage| replace:: ``$WECSIM/tutorials``
+.. _tutorialspage: https://github.com/WEC-Sim/WEC-Sim/tree/master/tutorials
```

Comment:
Is this intended to replace all local path references with pointers to the GitHub repository?

Reply:
No, I was just using it in my note. Although it is also an example of how you might have a code block act as a link.